scripts/query_task_with_image.sh

```diff
@@ -0,0 +1,8 @@
+node dist/index.js querytask \
```

Comment:
We do not need these 2 scripts.
What I mean is we need add 2 new commands  in the cli.
One can get the image detail. One can get the user detail.
which use the function queryImage() and queryUser() in the query.ts